material_components_ios.bzl

```diff
@@ -253,20 +255,48 @@ def mdc_snapshot_test(
       size = size,
       **kwargs)
 
+def mdc_ci_config_setting():
+    """Config setting for mdc continuous integration, e.g. --define ci_mode=kokoro"""
+    native.config_setting(
+        name = "kokoro",
+        values = {"define": "ci_mode=kokoro"},
+    )
+    native.config_setting(
+        name = "autobot",
+        values = {"define": "ci_mode=autobot"},
+    )
```

Comment:
These can likely be declared at the top level of the file so that you don't need a function.

Reply:
The `native` function can't be directly called in `bzl` file. I tried it and it say it needs to be moved into a macro.